apps/functions-subscription/src/adapters/azure/functions/middleware.ts

```diff
@@ -34,6 +34,18 @@ export const parsePathParameter =
       E.mapLeft(({ message }) => new H.HttpBadRequestError(message)),
     );
 
+/**
+ * Parses a specific query parameter of an HTTP request using the provided schema.
+ */
+export const parseQueryParameter =
```

Comment:
**suggestion**: add unit test for this function

Reply:
Added tests in eff99bbaf41355bc1a98ee131ab9a4d2694136f4